clang/lib/Sema/SemaDeclCXX.cpp

```diff
@@ -4368,7 +4368,7 @@ bool Sema::DiagRedefinedPlaceholderFieldDecl(SourceLocation Loc,
   Diag(Loc, diag::err_using_placeholder_variable) << Name;
   for (DeclContextLookupResult::iterator It = Found; It != Result.end(); It++) {
     const NamedDecl *ND = *It;
-    if (ND->getDeclContext() != ND->getDeclContext())
+    if (ND->getDeclContext() != ClassDecl->getDeclContext())
```

Comment:
This looks like a functional change to me; if no tests broke, we should probably figure out a test to add for this change.

Reply:
I am not familiar with this part of code to add a testcase, and not even sure it is right fix. I will just remove this changes from this PR.